Common/src/geometry/CPhysicalGeometry.cpp

```diff
@@ -29,10 +29,20 @@
 #include "../../include/adt_structure.hpp"
 #include "../../include/toolboxes/printing_toolbox.hpp"
 #include "../../include/toolboxes/CLinearPartitioner.hpp"
-#include "../../include/CSU2ASCIIMeshReaderFVM.hpp"
-#include "../../include/CCGNSMeshReaderFVM.hpp"
-#include "../../include/CRectangularMeshReaderFVM.hpp"
-#include "../../include/CBoxMeshReaderFVM.hpp"
+#include "../../include/geometry/meshreader/CSU2ASCIIMeshReaderFVM.hpp"
+#include "../../include/geometry/meshreader/CCGNSMeshReaderFVM.hpp"
+#include "../../include/geometry/meshreader/CRectangularMeshReaderFVM.hpp"
+#include "../../include/geometry/meshreader/CBoxMeshReaderFVM.hpp"
+
+#include "../../include/geometry/primal_grid/CPrimalGrid.hpp"
```

Comment:
Instead of primal/dual, another option is to divide up the files in the subfolders would be between FVM and FEM. I think either would work. Also, there is some subset of files needed by the FEM-based structural solver that is not really reflected in the naming here (we also have the files in elements/ which are FEA primal elements, for instance), which could be confusing.

Reply:
Those FEA elements are used more like numerics classes.

Reply:
I also thought about dividing it up between FVM and FEM. However, I would say that at the moment they are not independent enough (e.g. CPrimalGridFEM is derived from CPrimalGrid). In order to change this, a restructuring of the CGeometry classes is necessary. Moving these files around should not lead to a lot of problems since they are rarely changed.

Reply:
To phrase it differently: the structure proposed here reflects how it is implemented right now.